test/integration/test_pkg_signing.py

```diff
@@ -13,7 +14,7 @@ def test_pkg_sign():
     signer = textwrap.dedent(r"""
         import os
 
-        def sign(ref, artifacts_folder, signature_folder):
+        def sign(ref, artifacts_folder, signature_folder, **kwargs):
```

Comment:
According to the documentation: https://docs.conan.io/2/reference/extensions/package_signing.html